message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+      </entry>
+    </enum>
+    <enum name="POI_CLASSIFICATION">
+      <description>
+        Point-of-Interest classification.
+        Provides default classes (0 to 2147483647 reserved), but implementation-specific classes can be considered instead on the field where this classification is set.
+      </description>
+      <entry value="0" name="POI_CLASSIFICATION_UNKNOWN">
+        <description>Unknown classification.</description>
+      </entry>
+      <entry value="1" name="POI_CLASSIFICATION_GEO_MARKER">
+        <description>Geolocation marker.</description>
+      </entry>
+      <entry value="2" name="POI_CLASSIFICATION_LANDING_PAD">
+        <description>Landing pad.</description>
+      </entry>
+      <entry value="3" name="POI_CLASSIFICATION_CLEAR_LANDING_SPOT">
+        <description>Detected suited landing spot.</description>
+      </entry>
+      <entry value="4" name="POI_CLASSIFICATION_BUILDING">
+        <description>Building.</description>
+      </entry>
+      <entry value="5" name="POI_CLASSIFICATION_PERSON">
+        <description>Person.</description>
+      </entry>
+      <entry value="6" name="POI_CLASSIFICATION_VEHICLE">
+        <description>Vehicle.</description>
+      </entry>
+      <entry value="7" name="POI_CLASSIFICATION_INGRESS_PORTAL">
+        <description>Ingress portal.</description>
+      </entry>
+    </enum>
+  </enums>
+  <messages>
+    <message id="238" name="POI_REPORT">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+      <description>Reports information for a detected Point-of-Interest (POI).
+        This can be either set by an operator (through MAV_CMD_SET_POI_FOCUS) or through onboard software or external processes.
+        The receiving system should track the POI being reported based on its UID + time of detection + time of update.
+        Detection time happens once for the same UID, while time of update changes when a specific metadata of that POI gets changed (e.g. position).
+        The time of update should be changed on the sending system, based on the determined data in regards to that specific POI.
+        So, POIs that are received again should be updated if the time of update has changed.
+        Note: The sending system should repeat the current POIs at a fixed default rate of 2Hz to keep the protocol stateless). The fixed rate though can be set by the receiver using MAV_CMD_SET_MESSAGE_INTERVAL, which is advised for POIs that are not static or for which the state updates are high - decision on the rates of some specific POIs is at the implementers consideration.
+      </description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
+      <field type="uint64_t" name="time_utc_detected" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the POI detection, in UTC. 0 for unknown.</field>
```

Comment:
This is time since UNIX epoch, so an absolute time. That's good. Can we assume the emitter has a reliable source of the UNIX epoch?

Reply:
> Can we assume the emitter has a reliable source of the UNIX epoch?

Yes.

> I know you're going to emit 0 if unknown, what I'm wondering is whether lack of that information is critical?

This message will never be sent without a detection time.